pinot-core/src/test/java/org/apache/pinot/core/query/pruner/ColumnValueSegmentPrunerTest.java

```diff
@@ -148,17 +158,18 @@ public void testBloomFilterInPredicatePruning() {
     when(indexSegment.getDataSource("column")).thenReturn(dataSource);
     // Add support for bloom filter
     DataSourceMetadata dataSourceMetadata = mock(DataSourceMetadata.class);
-    BloomFilterReader bloomFilterReader = mock(BloomFilterReader.class);
+    BloomFilterReader bloomFilterReader = new BloomFilterReaderBuilder()
+        .put("1")
```

Comment:
Would you prefer

```java
BloomFilterReader bloomFilterReader = new BloomFilterReaderBuilder().put("1").put("2").put("3").put("5").put("7").put("21")
        .build();
```

instead? If that's the case I think that it is significantly more difficult to read.

Reply:
We actually relaxed the formatter on builders because there was consensus that a line per method was easier to read. It's only a nit so you don't have to do this, it's just a suggestion.

Reply:
I usually put minor/nit for optional comment, and you may ignore them. I'm good with both format, and I put one comment here because I remember Richard likes the compact way so that we can get more content when reading the code on a small screen :-P